Add node interface documentation (#183)
ChrisScianna/ROS-Underwater-RnD: mission_control/README.md

`Mission control` uses the [BehaviorTree.CPP](https://www.behaviortree.dev/) library for mission definition. A Behavior Tree (BT) __is a way to structure the switching between different tasks__. This is nothing more than a mechanism to invoke callbacks at the right time under the right conditions.

## Features
- Missions can be designed using [Groot](https://github.com/BehaviorTree/Groot)
- Missions are described as behavior trees in XML

> - Abort missions and go to surface in the event of unexpected anomalies (e.g. system faults)

Reviewer: @patapufate hmm, did you mean something like:
```suggestion
- Safe mission aborts -- vehicle is commanded to surface in the event of unexpected anomalies (e.g. system faults)
```
?

Author: Fixed
